frontend/src/HelperFunctions/customAxios.js

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+import history from "./customHistory";
+
+const axiosInstance = axios.create();
+
+axiosInstance.interceptors.response.use(
+  function (response) {
+    return response;
+  },
+  function (er) {
+    if (axios.isAxiosError(er)) {
```

Comment:
Wonder if this is a good thing to explicitly log, so we know why we've been navigated away?